src/gui/usbipd_gui.rs

```diff
@@ -75,37 +77,22 @@ pub struct UsbipdGui {
 
     // Tray icon
     #[nwg_control(icon: Some(&data.app_icon), tip: Some("WSL USB Manager"))]
-    #[nwg_events(OnContextMenu: [UsbipdGui::show_tray_menu], MousePressLeftUp: [UsbipdGui::show])]
+    #[nwg_events(OnContextMenu: [UsbipdGui::show_menu_tray(RC_SELF)], MousePressLeftUp: [UsbipdGui::show(RC_SELF)])]
     tray: nwg::TrayNotification,
 
-    // Tray menu
-    #[nwg_control(parent: window, popup: true)]
-    menu_tray: nwg::Menu,
-
-    #[nwg_control(parent: menu_tray, text: "Open")]
-    #[nwg_events(OnMenuItemSelected: [UsbipdGui::show])]
-    menu_tray_open: nwg::MenuItem,
-
-    #[nwg_control(parent: menu_tray)]
-    menu_tray_sep: nwg::MenuSeparator,
-
-    #[nwg_control(parent: menu_tray, text: "Exit")]
-    #[nwg_events(OnMenuItemSelected: [UsbipdGui::exit])]
-    menu_tray_exit: nwg::MenuItem,
-
     // File menu
     #[nwg_control(parent: window, text: "File", popup: false)]
     menu_file: nwg::Menu,
 
     #[nwg_control(parent: menu_file, text: "Refresh")]
-    #[nwg_events(OnMenuItemSelected: [UsbipdGui::refresh])]
+    #[nwg_events(OnMenuItemSelected: [UsbipdGui::refresh(RC_SELF)])]
     menu_file_refresh: nwg::MenuItem,
 
     #[nwg_control(parent: menu_file)]
     menu_file_sep1: nwg::MenuSeparator,
 
     #[nwg_control(parent: menu_file, text: "Exit")]
-    #[nwg_events(OnMenuItemSelected: [UsbipdGui::exit])]
+    #[nwg_events(OnMenuItemSelected: [UsbipdGui::exit()])]
```

Comment:
Revert, not needed.

Reply:
It's not needed, but exit() does not need a reference to self, so why expose it?